### PR TITLE
chore: move helm-backstage repo to redhat-developer/rhdh-chart 

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

chore: move https://github.com/janus-idp/helm-backstage -> https://github.com/redhat-developer/rhdh-chart [ci skip]

Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/RHIDP-1020

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: showcase-docs/monitoring-and-logging.md [hunk 1]
```diff
@@ -2,20 +2,20 @@
 
 The Backstage Showcase provides a `/metrics` endpoint that provides Prometheus metrics about your backstage application. This endpoint can be used to monitor your backstage instance using Prometheus and Grafana.
 
-When deploying Backstage Showcase onto a kubernetes cluster with the [Janus Helm chart](https://github.com/janus-idp/helm-backstage), monitoring and logging for your Janus instance can be configured using the following steps.
+When deploying Backstage Showcase onto a kubernetes cluster with the [Janus Helm chart](https://github.com/redhat-developer/rhdh-chart), monitoring and logging for your Janus instance can be configured using the following steps.
```
Comment (kim-tsao) [inline showcase-docs/monitoring-and-logging.md:5]: Are we still keeping the name Janus?

**Review by kim-tsao — APPROVED** [review]

File: showcase-docs/monitoring-and-logging.md (hunk 1 above)
Comment (nickboldt) [inline showcase-docs/monitoring-and-logging.md:5]:
```suggestion
When deploying Backstage Showcase onto a kubernetes cluster with the [RHDH Helm chart](https://github.com/redhat-developer/rhdh-chart), monitoring and logging for your Janus instance can be configured using the following steps.
```

File: showcase-docs/monitoring-and-logging.md [hunk 2]
```diff
@@ -2,20 +2,20 @@
...
-When deploying Backstage Showcase onto a kubernetes cluster with the [Janus Helm chart](https://github.com/janus-idp/helm-backstage), monitoring and logging for your Janus instance can be configured using the following steps.
+When deploying Backstage Showcase onto a kubernetes cluster with the [RHDH Helm chart](https://github.com/redhat-developer/rhdh-chart), monitoring and logging for your Janus instance can be configured using the following steps.
 
 ## Prerequisites
 
 - Kubernetes 1.19+
 - Helm 3.2.0+
 - PV provisioner support in the underlying infrastructure
-- The [Janus Helm chart repositories](https://github.com/janus-idp/helm-backstage#installing-from-the-chart-repository) have been added
+- The [Janus Helm chart repositories](https://github.com/redhat-developer/rhdh-chart#installing-from-the-chart-repository) have been added
```
Comment (nickboldt) [inline showcase-docs/monitoring-and-logging.md:12]:
```suggestion
- The [RHDH Helm chart repositories](https://github.com/redhat-developer/rhdh-chart#installing-from-the-chart-repository) have been added
```

File: showcase-docs/monitoring-and-logging.md [hunk 3]
```diff
@@ -2,20 +2,20 @@
 
 The Backstage Showcase provides a `/metrics` endpoint that provides Prometheus metrics about your backstage application. This endpoint can be used to monitor your backstage instance using Prometheus and Grafana.
 
-When deploying Backstage Showcase onto a kubernetes cluster with the [Janus Helm chart](https://github.com/janus-idp/helm-backstage), monitoring and logging for your Janus instance can be configured using the following steps.
+When deploying Backstage Showcase onto a kubernetes cluster with the [RHDH Helm chart](https://github.com/redhat-developer/rhdh-chart), monitoring and logging for your Janus instance can be configured using the following steps.
```
Comment (nickboldt) [inline showcase-docs/monitoring-and-logging.md:5]:
```suggestion
When deploying Backstage Showcase onto a kubernetes cluster with the [RHDH Helm chart](https://github.com/redhat-developer/rhdh-chart), monitoring and logging for your RHDH instance can be configured using the following steps.
```

File: showcase-docs/monitoring-and-logging.md [hunk 4]
```diff
@@ -2,20 +2,20 @@
...
 - PV provisioner support in the underlying infrastructure
-- The [Janus Helm chart repositories](https://github.com/janus-idp/helm-backstage#installing-from-the-chart-repository) have been added
+- The [RHDH Helm chart repositories](https://github.com/redhat-developer/rhdh-chart#installing-from-the-chart-repository) have been added
 
 ## Metrics Monitoring
 
 ### Enabling Metrics Monitoring on Openshift
 
-To enable metrics on Openshift, you will need to modify the `values.yaml` of the [Janus Helm chart](https://github.com/janus-idp/helm-backstage/blob/main/charts/backstage/values.yaml)
+To enable metrics on Openshift, you will need to modify the `values.yaml` of the [Janus Helm chart](https://github.com/redhat-developer/rhdh-chart/blob/main/charts/backstage/values.yaml)
```
Comment (nickboldt) [inline showcase-docs/monitoring-and-logging.md:18]:
```suggestion
To enable metrics on Openshift, you will need to modify the `values.yaml` of the [RHDH Helm chart](https://github.com/redhat-developer/rhdh-chart/blob/main/charts/backstage/values.yaml)
```

File: showcase-docs/monitoring-and-logging.md (hunk 1 above)
Comment (nickboldt) [reply showcase-docs/monitoring-and-logging.md]: nope, good catch. Pushed a few changes related to switching to RHDH name.

**Comment by nickboldt** [comment]
See also #946

**Review by schultzp2020 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**